src/components/connect/Graphics.vue

```diff
@@ -13,7 +13,7 @@
           id="pixelsPerDay"
```

Comment:
I appreciate how you kept this visual polish commit separate from some of the other commits earlier that did heavy restructuring.